src/masternode-payments.cpp

```diff
@@ -669,7 +669,7 @@ bool CMasternodePaymentWinner::IsValid(CNode* pnode, std::string& strError)
         // We don't want to print all of these messages, or punish them unless they're way off
         if (n > MNPAYMENTS_SIGNATURES_TOTAL * 2) {
             strError = strprintf("Masternode not in the top %d (%d)", MNPAYMENTS_SIGNATURES_TOTAL * 2, n);
-            LogPrintf("CMasternodePaymentWinner::IsValid - %s\n", strError);
+            LogPrint("masternode","CMasternodePaymentWinner::IsValid - %s\n", strError);
```

Comment:
post-merge: This should have been left as `LogPrintf()` as it results in a `Misbehaving` message.